docs/learn/calls.md

```diff
@@ -116,23 +224,29 @@ These common parameters provide a consistent way to control the behavior of LLM
 
     [`mirascope.core.anthropic.call_params`](../api/core/anthropic/call_params.md)
 
+    [`mirascope.core.azure.call_params`](../api/core/anthropic/azure.md)
+
     [`mirascope.core.cohere.call_params`](../api/core/cohere/call_params.md)
 
     [`mirascope.core.gemini.call_params`](../api/core/gemini/call_params.md)
 
     [`mirascope.core.groq.call_params`](../api/core/groq/call_params.md)
 
+    [`mirascope.core.litellm.call_params`](../api/core/litellm/call_params.md)
```

Comment:
nit: litellm doesn't have call params, it uses openai call params, so link the openai ones